nRF51/rbc_mesh/src/timer.c

```diff
@@ -99,8 +103,11 @@ void timer_event_handler(void)
         if (NRF_TIMER0->EVENTS_COMPARE[i])
         {
             NRF_TIMER0->EVENTS_COMPARE[i] = 0;
+#ifdef RACE_CONDITION_SOLVED
```

Comment:
Should this be reported as an issue? We've had some trouble here, and hope to change it next release. Have you had any bugs related to this?